pkg/account/model.go

```diff
@@ -0,0 +1,20 @@
+package account
+
+const ACCOUNT_PASSWD_LEN = 8
+
+type Account struct {
+	Email string `gorm:"primarykey"`
+
+	// Type   AcountType
+	Type   int
+	Name   string
+	Passwd string // Wait for encryption
+}
+
+type AcountType int
+
+const (
+	ACCOUNT_TYPE_PATIENT AcountType = iota
```

Comment:
And check the linter before you open a pr. You can make it a draft if it is not ready for review.